extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -231,7 +235,7 @@ protected static function onLoggableUserIPData( IContextSource $context, array $
 
 		$dbw = wfGetDB( DB_MASTER );
 		$cuc_id = $dbw->nextSequenceValue( 'cu_changes_cu_id_seq' );
-		$rcRow = array(
+		$rcRow = [
 			'cuc_id'         => $cuc_id,
 			'cuc_page_id'    => $data['pageid'], // may be 0
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)